package.json

```
@@ -2,7 +2,7 @@
   "name": "becca_lyria",
   "author": "Nicholas Carrigan",
   "main": "./prod/src/main.js",
-  "version": "14.0.3",
+  "version": "14.0.4",
```

```suggestion
  "version": "14.0.3",
```

You'll want to revert this manual change, then run `npm version patch` to have NPM generate a commit+tag for you and push that up.

---

Fixed, thanks. 🙂